bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,559 @@
+
+	<channel-group-type id="position">
+		<label>Position</label>
+		<description>Location information</description>
+		<category>MoveControl</category>
+		<channels>
+			<channel id="position" typeId="position"/>
+			<channel id="heading" typeId="heading"/>
+			<channel id="type" typeId="type"/>
+			<channel id="signal" typeId="signal"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="various">
+		<label>Various</label>
+		<description>Miscellaneous information</description>
+		<channels>
+			<channel id="lastUpdated" typeId="lastUpdated"/>
+			<channel id="privacy" typeId="privacy"/>
+			<channel id="belt" typeId="belt"/>
+			<channel id="emergency" typeId="emergency"/>
+			<channel id="service" typeId="service"/>
+			<channel id="preconditioning" typeId="preconditioning"/>
+			<channel id="preconditioningFailure" typeId="preconditioningFailure"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="current">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Current</label>
+		<description>Electrical current</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage</label>
+		<description>Voltage</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Temperature</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="daytime">
+		<item-type>Contact</item-type>
+		<label>Daytime</label>
+		<description>Enabled if it is daytime</description>
+		<category>Sun</category>
+	</channel-type>
+
+	<channel-type id="doorLock">
+		<item-type>String</item-type>
+		<label>Door lock</label>
+		<description>Door lock state</description>
+		<category>Door</category>
+		<state readOnly="true">
+			<options>
+				<option value="Unlocked">Unlocked</option>
+				<option value="Locked">Locked</option>
+				<option value="SuperLocked">Super Locked</option>
+				<option value="DriverDoorUnlocked">Driver Door Unlocked</option>
+				<option value="CabinDoorsUnlocked">Cabin Doors Unlocked</option>
+				<option value="CargoDoorsLocked">Cargo Doors Locked</option>
+				<option value="CargoDoorsUnlocked">Cargo Doors Unlocked</option>
+				<option value="RearDoorsLocked">Rear Doors Locked</option>
+				<option value="RearDoorsUnlocked">Rear Doors Unlocked</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="doorOpen">
+		<item-type>Contact</item-type>
+		<label>Door open</label>
+		<description>Door is open</description>
+		<category>Door</category>
+	</channel-type>
+
+	<channel-type id="ignition">
+		<item-type>String</item-type>
+		<label>Ignition</label>
+		<description>Ignition state</description>
+		<category>Motion</category>
```

Comment:
See above. Not listed. Please check all.

Reply:
It is a channel category, I believe: https://www.openhab.org/docs/concepts/categories.html#properties

Reply:
You're right, but I don't see the relation to the Motion category. This category is more intended for motion or presence detectors.